dartagnan/src/main/java/com/dat3m/dartagnan/wmm/definition/ExclusivePairs.java

```diff
@@ -0,0 +1,17 @@
+package com.dat3m.dartagnan.wmm.definition;
+
+import com.dat3m.dartagnan.wmm.Definition;
+import com.dat3m.dartagnan.wmm.Relation;
+import com.dat3m.dartagnan.wmm.RelationNameRepository;
+
+public class ExclusivePairs extends Definition {
```

Comment:
Rename to `LXSXPairs` or similar. The name "ExclusivePairs" is too misleading.

Reply:
`ReserveExclusive` or `ReserveConditional` could be more intuitive.

Reply:
I would go with a name that is either close to the standard CAT names, close to our IR, or close to the standard terms, i.e., something that is easy to google for. For example `LL/SC` is the most common abbreviation for it, but in ARM and our IR the abbreviation LXSX makes also sense. `ReservedConditionalPairs`/`ReservedExclusivePairs` is also ok (I would add the word `Pairs`).